content/doc/tutorials/building-a-python-app-with-pyinstaller.adoc

```diff
@@ -0,0 +1,486 @@
+---
+layout: documentation
+title: Using Jenkins to build a Python application with PyInstaller
```

Comment:
Now that we have a few tutorials, our poorly designed (by yours truly) sidebar doesn't really accomodate this longer text very well.

I suggest changing the titles to be a bit more imperative with just "Build a Python app with PyInstaller" or something along those lines

Reply:
Ah - I was almost going to defend the gerund but I think the current trend is moving away.

I'll update the titles of all the tutorials then to being with the imperative form of the verb.

Reply:
Okay - this is all done and redirect stubs have been implemented. I even updated links to the updated tutorial URLs within jenkins.io itself.

The slightly unforseen consequence of this is that 2 of the sample repository names still begin with gerunds - i.e. 'creating-...' and 'building-...'. Will hopefully deal with this in a subsequent PR.